skynet/modules/stt/streaming_whisper/state.py

```diff
@@ -28,20 +28,19 @@ def __init__(
         # before starting to drop them
         add_max_silent_chunks: int = 1,
         final_after_x_silent_chunks: int = 2,
-        force_final_duration_threshold: int = 29,
+        force_final_duration_threshold: int = 15,
         perform_final_after_silent_seconds: float = 0.8,
     ):
-        self.transcription_id = str(uuid6.uuid7())
-        self.transcription_ts = utils.now()
-        self.STARTED_STREAMING = utils.now()
+        self.transcription_id = str(utils.uuid_from_time(utils.now()))
+        self.received_timestamp_from_chunk = utils.now()
+        self.working_audio_starts_at = 0
         self.participant_id = participant_id
         self.silent_chunks = 0
+        self.chunk_count = 0
         self.working_audio = b''
         self.lang = lang
-        self.all_final = ''
         self.long_silence = False
-        self.chunk_count = 0
-        self.user_time = 0.0
+        self.chunk_duration = 0.0
```

Comment:
Seeing a number of chunk related bookeeping we have, would it make sense to create a little `Chunk` class wrapping the binary data, with some helper methods perhaps?